### ISSUE TITLE
Multiple arrows on date selector

### ISSUE THREAD

**Issue by Saikat-Sinha** [StephenChou1017/react-big-scheduler#63]
It showed up after 0.2.4 release

![screen shot 2018-11-24 at 12 35 43 pm](https://user-images.githubusercontent.com/17276324/48965551-9cae0680-efe5-11e8-8821-20f0524a3f7a.png)

**Comment by StephenChou1017** [comment]
@Saikat-Sinha 
I've tried it myself, it seems normal, you may try [this demo](https://github.com/StephenChou1017/react-big-scheduler-demo)

**Comment by Saikat-Sinha** [comment]
The issue persists even in this demo.
It may be due to  https://github.com/ant-design/ant-design/issues/12258

In our case, https://github.com/StephenChou1017/react-big-scheduler/blob/master/src/less/antd-globals-hiding-hack.less

generates
```
 #RBS-Scheduler-root .anticon-right:before {
  content: "\e61f";
}
#RBS-Scheduler-root .anticon-left:before {
  content: "\e620";
}
```

**Comment by blucas** [comment]
I see the same issue using RBS 0.2.4 and using the [demo repository](https://github.com/StephenChou1017/react-big-scheduler-demo)

<img width="1020" alt="screen shot 2018-11-24 at 10 06 04 am" src="https://user-images.githubusercontent.com/1743698/48969767-58187000-efd1-11e8-9910-6d17d81a0954.png">

**Comment by nimesha95** [comment]
I'm facing the same issue too

**Comment by StephenChou1017** [comment]
OK, I'll try to reappear this issue and fix it.

**Comment by tgBryanBailes** [comment]
This is happening because ant-design changed the way that icons are rendered in 3.9. Prior to 3.9, icons used a font. Now, icons are SVG. `antd-globals-hiding-hack.css` reproduces the prior way to render fonts.

Temporary Solution: uninstall antd and install an earlier version:
`npm uninstall antd`
`npm install antd@3.8.4`

Permanent Solution: I will be submitting a PR shortly which changes the `package.json` to utilize antd>3.9.0 and also removes the unnecessary styles from `antd-globals-hiding-hack.css`.

**Comment by StephenChou1017** [comment]
Another question. Since the multiple arrows are caused by RBS importing another copy of antd global css, how about this solution: RBS still outputs the `antd-globals-hiding-hack.css`, but don't import it by default. Then

1. All web site pages are antd style compatible and use antd global CSS rules;
In this case, for we already have the global css in antd, so it's OK by default.

2. The web site pages have their own global CSS rules, only a few pages use the Scheduler component.
In this case, we don't want the antd global css rules, so we import the `antd-globals-hiding-hack.css` manually in the pages using RBS.

**Comment by pelagia** [comment]
@StephenChou1017
I am using your demo repo but I cannot view the events.
Do I need to set up anything else for style apart from importing the react-big-scheduler/lib/css/style.css ??

**Comment by tgBryanBailes** [comment]
@StephenChou1017 I think the issue is that you are running from your dev environment and not the build you are putting on NPM.

If I create a brand new project using the instructions at https://github.com/StephenChou1017/react-big-scheduler-demo, the result reproduces the issue. antd 3.10.9 and react-big-scheduler 0.2.4 are installed.

**Comment by StephenChou1017** [comment]
@pelagia 
Hi, you may create a brand new project using the instructions at [https://github.com/StephenChou1017/react-big-scheduler-demo](https://github.com/StephenChou1017/react-big-scheduler-demo), it should works now.

**Comment by StephenChou1017** [comment]
@tgBryanBailes 
Thanks for the prompt, I reproduce the issue eventually by creating a brand new project instead of just cloning the demo repo.

I have merged your PR and published a new 0.2.5 for antd 3.9.0. Thanks again:-)

**Comment by Saikat-Sinha** [comment]
Closing this issue as it got addressed at version 0.2.5
